containers/unit-tests/scenario/distcheck

```diff
@@ -27,21 +27,7 @@ cd _distcleancheck/cockpit-*
 ./configure
 make distclean
 ./configure
-make -j$(nproc)
-make install DESTDIR=destdir
 make check 2>&1 || {
     find -name test-suite.log | xargs cat
     exit 1
 }
-cd ../..
-
-# do another build in a different directory and ensure that it produces the
-# same output (in the name of reproducible builds)
-mkdir _otherdir
-tar -C _otherdir -xf cockpit-[0-9]*.tar.xz
-cd _otherdir/cockpit-*
-./configure
-make -j$(nproc)
-make install DESTDIR=destdir
-cd ../..
-diff --no-dereference -ur _distcleancheck/*/destdir _otherdir/*/destdir
```

Comment:
Sad to see this go.  Let's try and figure out another way to bring it back later, possibly by passing the necessary CFLAGS as part of the `./configure` invocation.

Reply:
Right, that's a good idea. Once we figure out how Debian/Fedora builds already handle this, we can put that back here.